arch/models.py

```diff
@@ -46,6 +49,41 @@ def get_absolute_url(self):
     def get_statement(self) -> Optional[str]:
         return get_disk_statement_from_puid(self.puid)
 
+    @property
+    def niceness(self) -> float:
+        total: int = 0
+
+        votes = self.vote_set.all()
+
+        if len(votes) > 0:
+            for vote in votes:
+                total += vote.niceness
+
+            total /= len(votes)
+        else:
+            return None
+
+        return round(total, 2)
+
+
+class Vote(models.Model):
+    user = models.ForeignKey(
+        User,
+        help_text="User who voted for this problem.",
+        on_delete=models.CASCADE,
+    )
+
+    problem = models.ForeignKey(
+        Problem,
+        on_delete=models.CASCADE,
+        help_text="The container of the current vote.",
+    )
+
+    niceness = models.FloatField(
+        help_text="A student submitted number from 0 to 10 used to indicate the approximate niceness of a problem.",
+        validators=[MaxValueValidator(50)],
```

Comment:
should be out of 10 right?

Reply:
shouldn't this be a small positive integer field actually? rather than a floatfield

right now -9000.1434 is a valid input